native/src/work_asset/construct.rs

```diff
@@ -98,12 +98,13 @@ impl WorkAsset {
     ) -> Result<WorkAsset> {
         Self::transform_parse(&mut parse, &mut blob, file_base)?;
 
-        let default_tag = extension::get_validated_extension_tag(&parse, default_tag)?;
+        let default = Tag::from("default");
```

Comment:
yeah... we should really clean up the idea of default variant as it's no longer a thing, but this is a fine workaround for now